metricflow/cli/main.py

```diff
@@ -62,9 +67,32 @@ def cli(cfg: CLIContext, verbose: bool) -> None:  # noqa: D
         )
 
         click.echo(
-            f"💡 Please update to version {result.available_version}, released {result.release_date} by running:\n\t$ pip install --upgrade {PACKAGE_NAME}",
+            f"💡 Please update to version {result.available_version}, released {result.release_date} by running:\n"
+            f"\t$ pip install --upgrade {PACKAGE_NAME}",
         )
 
+    # Cancel queries submitted to the DW if the user precess CTRL + c / process is terminated.
+    # Note: docs unclear on the type for the 'frame' argument.
+    def exit_signal_handler(signal_type: int, frame) -> None:  # type: ignore
+        if signal_type == signal.SIGINT:
+            click.echo("Got SIGINT")
+        elif signal_type == signal.SIGTERM:
+            click.echo("Got SIGTERM")
+            signal.valid_signals()
+        else:
+            # Shouldn't happen since this should ony be registered for SIGINT / SIGTERM.
+            click.echo(f"Got signal {signal_type}")
+
+        if (
+            signal_type in (signal.SIGINT, signal.SIGTERM)
+            and cfg.sql_client.sql_engine_attributes.cancel_submitted_queries_supported
+        ):
+            logger.info("Cancelling submitted queries")
```

Comment:
where would this log to?

Reply:
This would log to a file in `~/.metricflow/logs`.